types/node/test/readline.ts

```diff
@@ -76,8 +76,8 @@ const rl: readline.ReadLine = readline.createInterface(new stream.Readable());
     const x = 1;
     const y = 1;
 
-    readline.cursorTo(strm, x);
```

Comment:
This line was in place to test the change in #17684, which has now been undone.

Current Node.js still appears to accept `y` as an optional parameter, but they also never updated the docs. (Doesn't look like the PR was ever created.)

Reply:
I think we should start with a PR for node otherwise such mismatches will confuse people again and again. Maybe I find some time the next days to do this.

Reply:
For the record, I have just opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37630. Hope this helps!

Reply:
I created a PR for node: https://github.com/nodejs/node/pull/29128